src/relayer/index.ts

```diff
@@ -87,6 +96,16 @@ export async function runRelayer(_logger: winston.Logger, baseSigner: Signer): P
         logger.warn({ at: "Relayer#run", message: "Assuming active relayer role in degraded state", degraded });
       }
 
+      if (!inventoryInit && config.useInventoryManager) {
+        const inventoryState = await getInventoryState(redis, inventoryClient.getInventoryCacheKey());
+        if (inventoryState) {
+          inventoryClient.import(inventoryState);
+          inventoryInit = true;
+        } else {
+          logger.error({ at: "Relayer#run", message: "No inventory state found in cache" });
+        }
```

Comment:
Minor - would be nice to log the key used here.

```suggestion
        const key = inventoryClient.getInventoryCacheKey();
        const inventoryState = await getInventoryState(redis, key);
        if (inventoryState) {
          inventoryClient.import(inventoryState);
          inventoryInit = true;
        } else {
          logger.error({ at: "Relayer#run", message: "No inventory state found in cache.", key });
        }
```